osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs

```diff
@@ -852,7 +853,8 @@ public async Task TestUpdateBeatmapFile()
                     var manager = osu.Dependencies.Get<BeatmapManager>();
 
                     string temp = TestResources.GetTestBeatmapForImport();
-                    await osu.Dependencies.Get<BeatmapManager>().Import(temp);
+
+                    osu.Dependencies.Get<BeatmapManager>().Import(temp).WaitSafely();
```

Comment:
Not sure I follow this one. The old version looked better if anything, because it had no sync waits at all...?

Reply:
It's a slightly unfortunate one. If a test method has the `async` keyword attached, it will be run from the thread pool, otherwise it won't.

I intentionally removed the `async` keyword here to allow this test to run without throwing an exception. The issue is on the access of `working.Beatmap`. I think we eventually want to change that property itself to not do a weird async wait in the background, but for now this was the easiest way of getting things to pass.